swap/src/protocol/alice/event_loop.rs

```diff
@@ -172,7 +185,12 @@ where
                 next_transfer_proof = self.send_transfer_proof.next() => {
                     match next_transfer_proof {
                         Some(Ok((peer, transfer_proof))) => {
-                            self.swarm.send_transfer_proof(peer, transfer_proof);
+                            let result = self.swarm.send_transfer_proof(peer, transfer_proof);
+
+                            if let Err(transfer_proof) = result {
+                                tracing::debug!(%peer, "No active connection to peer, buffering transfer proof");
```

Comment:
🙃 I'd say this is a `warn` level given that Bob would always reconnect this should not happen.